File: Lab 1/C#/main.cs
using System;

namespace StringExample
{
    class Program
    {
        static void Main(string[] args) 
        {
            Console.Write("enter line: ");
            string input = Console.ReadLine(); // Передається в StringHandler

            StringHandler stringHandler = new StringHandler(input);

            Console.WriteLine($"entered line: {stringHandler.String()}");
            Console.WriteLine($"length line is: {stringHandler.Length()}");
            Console.WriteLine($"reverse line: {stringHandler.ReverseString()}");
        }
    }
}

File: Lab 1/C++/main.cpp
#include <iostream>
#include "String.h"
using namespace std;

int main() {
    string userInput;

    cout << "enter line: ";
    getline(cin, userInput);

    String myString(userInput);

    cout << "entered line: " << myString.getString() << endl;
    cout << "length line is: " << myString.getLength() << endl;
    cout << "reverse line: " << myString.reverseString() << endl;

    return 0;
}
